formula/src/main/java/com/instacart/formula/internal/FormulaManagerImpl.kt

```diff
@@ -176,17 +173,16 @@ internal class FormulaManagerImpl<Input, State, RenderModel>(
         @Suppress("UNCHECKED_CAST")
         val manager = children
             .findOrInit(key) {
-                childManagerFactory.createChildManager(formula, input, transitionLock).apply {
-                    setTransitionListener { message, isValid ->
-                        handleTransition(Transition(effects = message), !isValid)
-                    }
+                val childTransitionListener = TransitionListener { effects, isValid ->
+                    handleTransition(Transition(effects = effects), !isValid)
                 }
+                childManagerFactory.createChildManager(formula, input, transitionLock, childTransitionListener)
             }
             .requestAccess {
-                throw java.lang.IllegalStateException("There already is a child with same key: $key. Use [key: String] parameter.")
-            } as FormulaManager<ChildInput, ChildState, ChildRenderModel>
+                throw java.lang.IllegalStateException("There already is a child with same key: $key. Use [key: Any] parameter.")
```

Comment:
I think we can remove the qualification and just have it be `IllegalStateException`, right?

Reply:
Not sure why it is like that in the first place.